recipe/meta.yaml

```
@@ -5,29 +5,38 @@
 # below since the only purpose of that command is to trigger the installation of the
 # dependencies.
 
-{% set version = "1.2.4" %}
+{% set name = "glueviz" %}
+{% set version = "1.4.0" %}
 
 package:
-  name: glueviz
-  version: {{version}}
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d3a6d2c845028ba8230bbc6130e09058206f4d6ad647b774a8f38f88ab6e53fc
 
 build:
   number: 0
-  # Skipping unnecessary linux platforms that don't need viz.
-  skip: True  # [py<37 or ppc64le or s390x]
+  skip: True  # [py<310]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
+    - pip
     - python
+    - setuptools_scm
+    - setuptools >=30.3
```

https://github.com/glue-viz/glueviz/blob/v1.4.0/pyproject.toml#L5
```suggestion
    - setuptools >=61.2
```